### ISSUE TITLE
Is it possible to make a checkpoint without using InsightFace?

### ISSUE THREAD

**Issue by ynie** [InstantID/InstantID#111]
Since InsightFace is not commercial free, Could we use something else and continue enjoying InstantID?

**Comment by ResearcherXman** [comment]
For current released models, no. But we are working on other face encoders to support commercial usage.

**Comment by ethank852** [comment]
any other face encoders your team is looking at? Maybe I can help play around as well

**Comment by ynie** [comment]
Yes we would like to help as well. Let’s move away from InsightFace please


-
Steven

**Comment by ResearcherXman** [comment]
Do you guys have any suggestion?

**Comment by ynie** [comment]
Hmm what did the non-face id Ipadapter use?

**Comment by whiterose199187** [comment]
@ResearcherXman  is https://github.com/davidsandberg/facenet an option?

**Comment by julien-blanchon** [comment]
> For current released models, no. But we are working on other face encoders to support commercial usage.

Any update on that ?

**Comment by mars747** [comment]
Is there any interest in a non-commercial model kickstarter to at least get a commercially usable API? i.e. make a kickstarter campaign so that the backers can get a commercial model through standard API prices (comparable to stable diffusion model serving prices)?

**Comment by johndpope** [comment]
fyi - there's some technology here that surprised me how well it could do face swapping without insightface
https://github.com/ICTMCG/CSCS
it's using arcface - there's a trained model on top of resnet50. 

UPDATE
they're mentioning here to use mediapipe instead of insightface.
https://github.com/kijai/ComfyUI-LivePortraitKJ
